### DISCUSSION TITLE
The linking table field is not displayed.

### DISCUSSION THREAD

**Discussion by skorikovdo** [aminalaee/sqladmin#818] [Q&A] [answered]
I have a table `count_operation`, which is located in the `project` schema, it is connected to the machine table by the `machine_id` field. But when creating a record for the `count_operation` table, the `machine_id` field is missing

![pycharm](https://github.com/user-attachments/assets/405d4e8b-3956-43f5-a713-0d658dc1f1f6)
![p2](https://github.com/user-attachments/assets/f828d76e-d751-4dd4-b57d-02337cc8b30e)

![panel](https://github.com/user-attachments/assets/2d99afc5-5290-4e95-9ea3-5b84332bd1b0)

**Comment by aminalaee** [comment, marked as answer]
I think you need to define a `relationship` for the machine which uses the `machine_id` on the `count_operation`.
